docs/manual/external-tools.tex

```diff
@@ -960,12 +962,34 @@ \subsection{Adding a checker via the project properties\label{netbeans-project-p
 \end{enumerate}
 
 
+\subsection {Adding a checker using the Checker Framework Plugin\label{netbeans-plugin}}
+
+In order to run a checker using the NetBeans plugin, you should first
+install the plugin. To do so, select ``Tools'' and then ``Plugins''.
+In the Plugins window, click on the ``Settings'' tab, and then ``Add''.
+In the popup that appears, enter ``Checker Framework'' under ``Name'' and
+the URL where the Checker Framework plugin is hosted under ``URL''.
+
+Currently, the Checker Framework plugin is kept at \url{https://twlogger.github.io/catalog.xml}.
+
+After adding the Checker Framework plugin Update Center, go to the
+``Available Plugins'' tab, and search for ``Checker Framework'', select
+the checkbox, and click ``Install''.
+
+Once the Checker Framework is installed, right click on the project
+that you wish to run the checker on, and click ``Properties''. Then,
+select the checkboxes of the checkers that you wish to run, and press
+``Ok'' to apply the changes. If ``Compile on save'' is enabled (this can
+also be set in the Project Properties window), the results of the checker
+should appear automatically in the editor.
```

Comment:
The results did not appear automatically for me.  I had to build the project for the warnings to appear in the "Output" tab.  (The "Compile on save" option is enabled.) When they did I got the following warning:
```
warning: You do not seem to be using the distributed annotated JDK.  To fix the
  problem, supply this argument (first, fill in the "...") when you run javac:
    -Xbootclasspath/p:.../checker/dist/jdk8.jar
```
I selected the Nullness Checker and the Regex Checker.  The warnings I expect from those checkers appeared in the output as well.

Reply:
@twlogger and I met last week and looked into this. Netbeans 8.1 uses Java 8 for the presentation compiler, but Netbeans 8.2 apparently switched to a Java 9 presentation compiler. I didn't find a note on it in the NB release notes, but e.g. Lombok ran into similar issues: https://github.com/rzwitserloot/lombok/issues/1194

Therefore, the Checker Framework can run automatically on Netbeans 8.1, but not on Netbeans 8.2.
There is no simple solution to this until we support the Java 9 AST, but then NB 8.1 wouldn't work anymore.
Netbeans 8.2 can still run the Checker Framework manually and display the results in the error window. 

I will update this PR to document this issue. Then at some point we can provide an updated plugin that works with Netbeans 8.2 upwards.